package.json

```diff
@@ -1,10 +1,12 @@
 {
   "name": "aws-lambda-custom-node-runtime",
-  "version": "1.0.1",
+  "version": "1.1.0",
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "npm run cleanup; node index ",
```

Comment:
What is the purpose of this? This package is meant to be installed globally. So there shouldn't need to be a build script I don't think.

Reply:
I was running it locally before I realized that I could just use npm link.